reference/entry.json

```diff
@@ -670,6 +670,129 @@
+                "examples": {
+                  "Invalid data": {
+                    "value": {
+                      "type": "https://api.publiq.be/probs/body/invalid-data",
+                      "title": "Invalid body data",
+                      "status": 400,
+                      "schemaErrors": [
+                        {
+                          "jsonPointer": "/url",
+                          "error": "The required properties url is missing."
+                        }
+                      ]
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "401": {
+            "$ref": "#/components/responses/Unauthorized"
+          },
+          "403": {
+            "$ref": "#/components/responses/Forbidden"
+          },
+          "404": {
+            "$ref": "#/components/responses/EventNotFound"
+          }
+        },
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "type": "object",
+                "properties": {
+                  "url": {
+                    "type": "string",
+                    "format": "uri",
+                    "example": "https://www.youtube.com/?v=re34sf",
+                    "description": "The URL of the video to add. Currently only Youtube and Vimeo sources are allowed.",
+                    "pattern": "^http(s?)://(www.)?(youtube|vimeo)\\."
+                  },
+                  "copyrightHolder": {
+                    "type": "string",
+                    "minLength": 3,
+                    "example": "publiq",
+                    "description": "The copyright holder of the video source."
+                  }
+                },
+                "required": [
+                  "url"
+                ]
+              },
+              "examples": {
+                "Video from Youtube and with Copyright": {
+                  "value": {
+                    "url": "https://www.youtube.com/?v=re34sf",
+                    "copyrightHolder": "publiq"
+                  }
+                },
+                "Video from Vimeo": {
+                  "value": {
+                    "url": "https://www.vimeo.com/4dwe2"
+                  }
+                }
+              }
+            },
+            "application/xml": {
+              "schema": {
+                "type": "object",
+                "properties": {}
+              }
+            }
+          },
+          "description": "The new video to add to an event"
+        },
+        "description": "Add a video as a URL reference to an event\n\nThe video objects contains:\n\n* `url`: The full URL of the video. Currently only *Vimeo* and *Youtube* are supported as video source locations.\n* `copyrightHolder`: The copyright holder of the video material. Although this field is optional it is strongly recommended to add a reference to the entity owning the rights on the video material.",
```

Comment:
The first sentence in the description is still missing a full stop (`.`) like I mentioned in a previous comment. (Same for place operation).

Reply:
Thanks for adding the extra context by the way, very clear 👍

Reply:
Added a `.`